### ISSUE TITLE
Flying camera allows to explore locked regions

### ISSUE THREAD

**Issue by YohDeadfall** [voxeltycoon/bug-tracker#151]
There is no stable way to reproduce the bug, but mostly it's possible to observe locked regions switching to the flying camera mode. This led me to a question asked via an e-mail (about map size since I thought that it was limited).

**Build version**
0.77 2019-09-12 ITCH Pre Alpha

**Platform**
OS Name	Microsoft Windows 10 Pro
Version	10.0.19013 Build 19013

**Comment by victorleblais** [comment]
Just a precision about the map size: the map is generated region by region. A fixed number of region is generated around the ones you have unlocked.

When you buy a region, new ones are being generated on the border of the map to keep a minimum "distance to the border" of the map.

The map size is not limited, but only a small part of it is created to avoid putting unnecessary strain on your computer.

**Comment by YohDeadfall** [comment]
Sometimes the camera can fly over locked regions, but sometimes it's unable. Mostly it cannot, so looks like the ability to explore locked area is a bug.

**Comment by victorleblais** [comment]
Yes, it's a bug, you should not be able to use the free camera to do that, it should be limited like the normal camera where you "stick" to your region's borders.

**Comment by andrewpey** [comment]
It's possible to go outside of unlocked regions when you went to the Editor mode from Free camera view (enabled with F10). Probably this is the case?

**Comment by YohDeadfall** [comment]
No, the Editor mode is disabled as you can see. The proof on the attached screenshot was taken using the latest build.

![screenshot](https://user-images.githubusercontent.com/10694013/82118293-eca2ff00-977e-11ea-853c-fecab53778a2.png)

**Comment by andrewpey** [comment]
Thanks for the proof 😉
